src/components/pod/pod.test.js

```diff
@@ -13,6 +55,234 @@ const PodWithSoftDelete = () => (
+      divPodComponent()
+        .should("have.css", "height", height)
+        .and("have.css", "width", width);
+    });
+
+    it.each([
+      [
+        "primary",
+        "rgb(255, 255, 255) none repeat scroll 0% 0% / auto padding-box border-box",
+        "none",
+      ],
+      [
+        "secondary",
+        "rgb(242, 245, 246) none repeat scroll 0% 0% / auto padding-box border-box",
+        "none",
+      ],
+      [
+        "tertiary",
+        "rgb(237, 241, 242) none repeat scroll 0% 0% / auto padding-box border-box",
+        "none",
+      ],
+      [
+        "tile",
+        "rgb(255, 255, 255) none repeat scroll 0% 0% / auto padding-box border-box",
+        "rgba(2, 18, 36, 0.2) 0px 2px 3px 0px",
+      ],
+      [
+        "transparent",
+        "rgba(0, 0, 0, 0) none repeat scroll 0% 0% / auto padding-box border-box",
+        "none",
+      ],
+    ])(
+      "should check %s variant for Pod component",
+      (variant, color, boxShadow) => {
+        CypressMountWithProviders(<PodComponent variant={variant} />);
+        divPodComponent()
+          .should("have.css", "back-ground", color)
+          .and("have.css", "box-shadow", boxShadow);
+      }
+    );
+
+    it.each(specialCharacters)(
+      "should check title as %s for Pod component",
+      (title) => {
+        CypressMountWithProviders(<PodComponent title={title} />);
+        podTitle().should("have.text", title);
+      }
+    );
+    it.each(specialCharacters)(
+      "should check subtitle as %s for Pod component",
+      (subtitle) => {
+        CypressMountWithProviders(<PodComponent subtitle={subtitle} />);
+        podSubTitle().should("have.text", subtitle);
+      }
+    );
+
+    it.each(["left", "center", "right"])(
+      "should check when title alignment is %s for Pod component",
+      (alignTitle) => {
+        CypressMountWithProviders(<PodComponent alignTitle={alignTitle} />);
+        podTitle().should("have.css", "text-align", alignTitle);
+      }
+    );
+
+    it.each(specialCharacters)(
+      "should check footer as %s for Pod component",
+      (footerText) => {
+        CypressMountWithProviders(<PodComponent footer={footerText} />);
+        podFooter().should("have.text", footerText).and("be.visible");
+      }
+    );
+
+    it.each(["100px", "200px", "300px"])(
+      "should check when internalEditButton is %s for Pod component",
+      (height) => {
+        CypressMountWithProviders(<PodComponent height={height} />);
+        podComponent().should("have.css", "height", height);
+      }
+    );
+
+    it.each([
+      [false, "not.exist"],
+      [true, "be.visible"],
+    ])(
+      "should check when softDelete is %s for Pod component",
+      (boolVal, state) => {
+        CypressMountWithProviders(
+          <SoftDeletePodComponent softDelete={boolVal} />
+        );
+        podSoftDelete().should(state);
+      }
+    );
+
+    it.each([
+      [false, "77.5625px"],
```

Comment:
same is here with `"77.5625px"`. I think we need to use `integers` instead of `decimals`